### PR TITLE
V3.0.2

### PR DESCRIPTION
**Pull request by santhoshkvuda**
V3.0.2

### CODE REVIEW

**Review by palukuri — DISMISSED** [review]

**Review by niravg75 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 2023-07-24
+### Added
+- Multi Process Worker Support
```
Comment (kgvarun) [inline CHANGELOG.md:5]: adding this link for context will be helpful - https://docs.fluentd.org/deployment/multi-process-workers

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+- Custom Container Image for Fluentd using OL8-Slim as base Image.
```
Comment (kgvarun) [inline CHANGELOG.md:6]: This is a new image on container-registry right? we should put image tag?

File: README.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ Dashboards needs to be imported manually. Below is an example for importing Dash
 
 1. Download and configure [OCI CLI](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm) or open cloud-shell where OCI CLI is pre-installed. Alternative methods like REST API, SDK, Terraform etc can also be used.
 2. Find the **OCID** of the compartment, where the dashboards need to be imported.
-3. Download the dashboard JSONs from [here](logan/terraform/oke/modules/dashboards/dashboards_json/) (TBD).
+3. Download the dashboard JSONs from [here](terraform/modules/dashboards/dashboards_json/).
```
Comment (kgvarun) [inline README.md:141]: Can we close https://github.com/oracle-quickstart/oci-kubernetes-monitoring/pull/39 with the new jsons?

File: charts/common/templates/clusterrole.yaml [hunk 4]
```diff
@@ -23,6 +23,7 @@ rules:
       - batch
       - discovery.k8s.io
       - metrics.k8s.io
+      - storage.k8s.io
```
Comment (kgvarun) [inline charts/common/templates/clusterrole.yaml:26]: I suggest adding this to release notes that this version expands clusterrole permissions

File: logan/docker-images/v1.0/oraclelinux/8/entrypoint.sh [hunk 5]
```diff
@@ -1,6 +1,3 @@
 #!/usr/bin/env sh
 
-# Copyright (c) 2023, Oracle and/or its affiliates.
```
Comment (kgvarun) [inline logan/docker-images/v1.0/oraclelinux/8/entrypoint.sh:3]: Oracle License small text is required in all the files.

**Review by kgvarun — CHANGES_REQUESTED** [review]

**Comment by kgvarun** [comment]
@santhoshkvuda pls see review comments inline.

File: CHANGELOG.md (hunk 2 above)
Comment (santhoshkvuda) [reply CHANGELOG.md:6]: Nope, this is referring to custom container images (instructions for creating the custom container images).

File: CHANGELOG.md (hunk 1 above)
Comment (santhoshkvuda) [reply CHANGELOG.md]: Done

File: README.md (hunk 3 above)
Comment (santhoshkvuda) [reply README.md:141]: As per our discussion pulled in the change related to Query part only but not the visualisation.

File: charts/common/templates/clusterrole.yaml (hunk 4 above)
Comment (santhoshkvuda) [reply charts/common/templates/clusterrole.yaml:26]: Done

File: logan/docker-images/v1.0/oraclelinux/8/entrypoint.sh (hunk 5 above)
Comment (santhoshkvuda) [reply logan/docker-images/v1.0/oraclelinux/8/entrypoint.sh:3]: Done

**Comment by santhoshkvuda** [comment]
> @santhoshkvuda pls see review comments inline.

@kgvarun , Addressed all the applicable review comments. Please check.

**Review by kgvarun — APPROVED** [review]

**Review by kgvarun — APPROVED** [review]

**Review by palukuri — APPROVED** [review]

**Review by niravg75 — APPROVED** [review]

**Pull request merged**